hack/verify-shellcheck.sh

```diff
@@ -36,13 +36,8 @@ SHELLCHECK_IMAGE="koalaman/shellcheck-alpine:v0.6.0@sha256:7d4d712a2686da99d3758
 all_shell_scripts=()
 while IFS=$'\n' read -r script;
   do git check-ignore -q "$script" || all_shell_scripts+=("$script");
-done < <(find . -name "*.sh" \
-  -not \( \
-    -path ./_\*      -o \
-    -path ./.git\*   -o \
-    -path ./vendor\* -o \
-    \( -path ./third_party\* -a -not -path ./third_party/forked\* \) \
-  \))
+done < <(grep -irl '#!/bin/bash' . \
```

Comment:
might be a little better:
```suggestion
done < <(grep -irl '#!.*sh' . \
```

most shell interpreter binaries are `.*sh` (`sh`, `dash`, `zsh`, `bash` ...), we should also not be picky about what path they are in so shebangs like `#!/usr/bin/env bash` are caught 🙃